include/osgParticle/ParticleSystem

```diff
@@ -265,7 +265,7 @@ namespace osgParticle
           * for all graphics contexts. */
         virtual void releaseGLObjects(osg::State* state=0) const;
 
-        virtual osg::VertexArrayState* createVertexArrayStateImplementation(osg::RenderInfo& renderInfo) const;
```

Comment:
This file is missing some changes:
setNumParticlesToCreateHint
   // BEGIN VRV_PATCH
        // don't compute these each time, just store in class
        const osg::Vec3 _xAxis;
        const osg::Vec3 _yAxis;
        const osg::Vec3 _zAxis;
        const Particle::Shape _userShape;
        // END VRV_PATCH